src/app/screens/BoardsList/layout.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+import { ROUTES } from '../../../config/routes';
+import { propTypes as boardPropTypes } from '../../../redux/boards/reducer';
+import './styles.css';
+
+const BoardsList = ({ boards, onDeleteBoard }) => (
+  <ul>
+    {boards.map(board => (
+      <div className="board-card" key={board.id}>
```

Comment:
I'd extract this into a `Board` component, so we can create `() => onDeleteBoard(board.id)` once and not on every render.

Reply:
yes, thats right :+1: